lib/Convos/Core/Backend.pm

```diff
@@ -58,14 +58,12 @@ sub users {
 sub _setup {
   my $self = shift;
 
-  Scalar::Util::weaken($self);
   $self->on(
     connection => sub {
       my ($self, $connection) = @_;
       my $cid = $connection->id;
       my $uid = $connection->user->id;
 
-      Scalar::Util::weaken($self);
```

Comment:
I don't understand why `weaken()` is not required here and/or above. (Same change is in File.pm)